fadtk/fad_calculation.py

```diff
@@ -0,0 +1,21 @@
+from argparse import ArgumentParser
+
+from .fad import FrechetAudioDistance, log
```

Comment:
we'd better set a logger in this file separately.
```
log = logging.getLogger(__name__)

```

Reply:
'fad_calculation.py' seems to be added by Modan, but not used now.
I'll remove the file. Sorry for the confusion.